spec/models/item_spec.rb

```diff
@@ -30,9 +30,27 @@
     expect(item).to_not be_valid 
   end
 
-    it "is invalid with an empty string for description" do
+  it "is invalid with an empty string for description" do
     item.description = ""
     expect(item).to_not be_valid 
   end
+  it "is invalid with duplicate titles" do
+    item
+    item2 = Item.new(title: "Possum Stew", description: "Savory stew made with fresh possum", image: "www.possum.com", item_status_id: 1, price: 2.35)
+    expect(item2).to_not be_valid 
+  end
+
+  it "is invalid with duplicate titles in difference case" do
+    item
+    item2 = Item.new(title: "possum stew", description: "Savory stew made with fresh possum", image: "www.possum.com", item_status_id: 1, price: 2.35)
+    expect(item2).to_not be_valid 
+  end
+
+  it "is invalid with a price less than 0" do
+    item.price = -1
+    expect(item).to_not  be_valid
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.